src/gpuarray_array_blas.c

```diff
@@ -439,6 +439,23 @@ int GpuArray_rger(double alpha, GpuArray *X, GpuArray *Y, GpuArray *A,
   return err;
 }
 
+static inline int is_2d_contiguous(const GpuArray *a) {
```

Comment:
The name isn't clear to what it does. Currently, it always expect a 2d inputs. If you want to keep that name,use ndim-1 and ndim-2 for which dimensions you checks. Otherwise, rename.